spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -155,12 +171,19 @@ ZipkinSpanHandler zipkinSpanHandler(Reporter<Span> spanReporter) {
 	@ConditionalOnClass(ZipkinSpanExporter.class)
 	static class OpenTelemetryConfiguration {
 
+		@Bean
+		@ConditionalOnMissingBean(value = Span.class, parameterizedContainer = BytesEncoder.class)
+		@ConditionalOnEnabledTracing
```

Comment:
I'd remove that annotation from the bean method. The idea is only to disable the data-reporting components and leaving everything else untouched.